app/scripts/lib/experiments/grouping-rules/token-code.js

```diff
@@ -6,17 +6,24 @@
 
 const BaseGroupingRule = require('./base');
 const Constants = require('../../../lib/constants');
-const GROUPS = ['control', 'treatment-code', 'treatment-link'];
+const GROUPS_DEFAULT = ['control', 'treatment-code', 'treatment-link'];
+
 const ROLLOUT_CLIENTS = {
-  '3a1f53aabe17ba32': {
-    name: 'Firefox Add-ons',
-    rolloutRate: 0.0 // Rollout rate between 0..1
-  },
-  'dcdb5ae7add825d2': {
-    name: '123Done',
+  '37fdfa37698f251a': {
+    enableTestEmails: false,
+    groups: GROUPS_DEFAULT,
+    name: 'Lockbox Extension',
```

Comment:
Is the name just to help us as devs? I don't see it used anywhere.

Reply:
Correct, name is just used for devs.